script.js

```diff
@@ -0,0 +1,11 @@
+const getBtnLogin = document.querySelector('#btn-login');
+
+getBtnLogin.addEventListener('click', () => {
+  const getInputLogin = document.querySelector('#input-login');
+  const getInputSenha = document.querySelector('#input-senha');
+  if (getInputLogin.value !== 'tryber@teste.com' || getInputSenha.value !== '123456') {
```

Comment:
Amigos, para esse projeto, vamos adotar o uso dos parênteses para melhorar a legibilidade do código. Veja a sugestão:
```suggestion
  if ((getInputLogin.value !== 'tryber@teste.com') || (getInputSenha.value !== '123456')) {
```